NitroxServer/GameLogic/Players/PersistedPlayerData.cs

```diff
@@ -23,36 +23,33 @@ public class PersistedPlayerData
     public Optional<NitroxId>[] QuickSlotsBindingIds { get; set; } = Array.Empty<Optional<NitroxId>>();
 
     [DataMember(Order = 4)]
-    public List<EquippedItemData> EquippedItems { get; set; } = new List<EquippedItemData>();
+    public Dictionary<string, NitroxId> EquippedItems { get; set; } = new Dictionary<string, NitroxId>();
 
     [DataMember(Order = 5)]
-    public List<EquippedItemData> Modules { get; set; } = new List<EquippedItemData>();
-
-    [DataMember(Order = 6)]
     public ushort Id { get; set; }
 
-    [DataMember(Order = 7)]
+    [DataMember(Order = 6)]
```

Comment:
I think in general we want to avoid having changes to his data structures but as we are going to breaking change this release I think it should be fine